tests/test_bot.py

```diff
@@ -95,7 +95,7 @@ def to_camel_case(snake_str):
     return components[0] + "".join(x.title() for x in components[1:])
 
 
-@pytest.fixture(scope="module")
+@pytest.fixture(scope="class")
```

Comment:
Can give an explanation on which scopes you setteld for in the end and why? :D

Reply:
settled for `function` in the end since bot tests modify the message returned from this, making it best to return a new message for each test.